### ISSUE TITLE
A11y Code Sprint Berlin: Success Criterion 9.2.4.1 - make iframe title more meaningful

### ISSUE THREAD

**Issue by SarahEdding** [benjaminkott/bootstrap_package#1296]
# Feature Request

## Description
All frames should have a title attribute containing information about the frame's content. Currently, iframes with YouTube videos are only labelled with the video's title. 

## Solution
Add "Video on YouTube" to title value. 
e.g.: `<iframe src="..." title="Video on YouTube: What is TYPO3?"></iframe>`

**Comment by jwundermann** [comment]
@SarahEdding The prefix "Video on YouTube:" could be set in the content element in the title of the video. Is done.

**Comment by linawolf** [comment]
@alexakressmann I believe this one can be reviewed

**Comment by SarahEdding** [comment]
I've set new title in filelist for four exisiting videos. For Text&Media, this shows up correctly in the frontend. 
External media, however, embeds the YouTube-Player directly, not via FAL-Relation. 
These iframes do not have a title. **For compliance they need one.** 
Can this be fixed? If not, this page should be hidden before the offical a11y-audit.

**Comment by schmigotzki** [comment]
@alexakressmann Please assign to me

**Comment by schmigotzki** [comment]
The "External Media" content element will get a new field "External Media iframe Title". This must be filled out.
